examples/README.md

```diff
@@ -3,20 +3,80 @@ This folder contains an example of how to use the gr-wavelearner package.
 
 ## Files:
 
-- classifier_test.grc: Executable GRC file demonstrating how to use gr-wavelearner package for
-                       inference of a neural network
-- cuda_fft_demo.grc: Executable GRC file demonstrating how to use the cuFFT block
+- `gpu_fft_demo.grc`: Executable GRC file demonstrating how to use the cuFFT block
+- `classifier_test.grc`: Executable GRC file demonstrating how to use gr-wavelearner
+  package for inference of a neural network
+                       
+## gpu_fft_demo Instructions:
+Please see the [tutorial](https://docs.deepwavedigital.com/Tutorials/3_cufft.html) that
+Deepwave has put together for this example.
 
-## Instructions:
+## classifier_test Instructions:
+Please see the [tutorial](https://docs.deepwavedigital.com/Tutorials/inference.html)
+that Deepwave has put together for this example.
 
-1. Generate the PLAN file for your trained network. For examples on how to generate the PLAN file, see
-[here](https://github.com/deepwavedigital/airstack-examples/tree/master/python/inference)
+1. Generate the PLAN file for your trained network. As an example, you may generate a plan
+   file from the onnx file provided in our [airstack-examples](https://github.com/deepwavedigital/airstack-examples)
+   repository. See the **Creating a PLAN File* section below for instructions on this
+   example.
 2. Open GNU Radio Companion and open `classifier_test.grc`
-3. Open the **Inference Block** and select the `.plan` file created in Step 1.
-4. Execute the GRC flowgraph
+3. Open the **Inference Block** and select the PLAN file created in Step 1.
+4. Ensure that the settings of the **Inference Block** match the PLAN file.
+5. Execute the GRC flowgraph
 
-Note that the examples utilize a SoapyAIRT source block. However, these examples can be tailored to any type of SDR by simply swapping the source block with the appropriate block for your hardware.
+Note that the examples utilize a SoapyAIRT source block. However, these examples can be
+tailored to any type of SDR by simply swapping the source block with the appropriate block
+for your hardware.
 
+### Creating a PLAN File
```

Comment:
I'd recommend removing this whole section and simply referencing https://github.com/deepwavedigital/airstack-examples/blob/master/inference/README.md